front/lib/resources/string_ids.ts

```diff
@@ -104,6 +104,13 @@ export function getResourceIdFromSId(sId: string): ModelId | null {
   return sIdsRes.value.resourceId;
 }
 
+export function isResourceSId(
+  resourceName: ResourceNameType,
+  sId: string
+): boolean {
+  return sId.startsWith(RESOURCES_PREFIX[resourceName]);
```

Comment:
We should test that it starts with the prefix and the underscore to remove any chance of a collision on the prefix since _ is not a valid legacy sId character 👍